package.json

```diff
@@ -94,74 +111,24 @@
     "build-ee": "npm run clean && npm test && NODE_ENV='production' ./node_modules/.bin/gulp dist",
     "dist": "npm run build-ee",
     "prebuild-ce": "./scripts/validate-tests",
-    "build-ce": "npm run clean && npm test && NODE_ENV='production' ./node_modules/.bin/gulp dist",
-    "serve": "NODE_ENV='development' ./node_modules/.bin/gulp serve",
+    "build-ce": "npm run clean && npm test && cross-env NODE_ENV='production' ./node_modules/.bin/gulp dist",
+    "serve": "cross-env NODE_ENV='development' ./node_modules/.bin/gulp serve",
     "serve-notify": "export NOTIFY=true && npm run serve",
-    "testing": "NODE_ENV='testing' ./node_modules/.bin/gulp serve",
+    "testing": "cross-env NODE_ENV='testing' ./node_modules/.bin/gulp serve",
+    "lint-plugins": "PLUGINS=$DCOS_UI_PLUGINS; ./node_modules/.bin/eslint \"$PLUGINS/**/*.js\" --quiet",
     "lint": "./node_modules/.bin/eslint '**/*.js' --quiet",
-    "livereload": "NODE_ENV='development' ./node_modules/.bin/gulp livereload",
+    "livereload": "cross-env NODE_ENV='development' ./node_modules/.bin/gulp livereload",
     "livereload-notify": "export NOTIFY=true && npm run livereload",
-    "test": "./node_modules/.bin/gulp copy:external-plugins && npm run lint && jest"
+    "test": "npm run lint && npm run lint-plugins && node jest/genConfig.js && NODE_PATH=node_modules jest --config=jest/config.json --no-cache",
+    "jest": "node jest/genConfig.js && NODE_PATH=node_modules jest --config=jest/config.json --no-cache",
```

Comment:
@mlunoe Our npm scripts will choke on Windows. However, I think everyone seems to be in favor of only providing support for build requirements as they are needed. - From `cross-env` ![image](http://cl.ly/0w1p0Y3E1m2o/Image%202016-04-27%20at%209.36.53%20AM.png)